index.src.html

```diff
@@ -17,96 +17,99 @@
 
-- On the phone:
-    * User goes to example.com in the browser and signs in to an existing account using whatever method they have been using
+### Registration (<a>embedded authenticator</a> mode) ### {#registration-embedded}
+
+- On a phone:
+    * User navigates to example.com in a browser and signs in to an existing account using whatever method they have been using
         (possibly a legacy method such as a password), or creates a new account.
     * The phone prompts, "Do you want to register this device with example.com?"
     * User agrees.
-    * The phone prompts the user for a previously configured authorization gesture (PIN, biometric, etc.); the user provides
+    * The phone prompts the user for a previously configured <a>authorization gesture</a> (PIN, biometric, etc.); the user provides
         this.
     * Website shows message, "Registration complete."
 
 
-## Authentication (<a>external authenticator</a> mode) ## {#authentication-external}
+### Authentication (<a>external authenticator</a> mode) ### {#authentication-external}
 
-- On the laptop:
-    * User goes to example.com in browser, sees an option "Sign in with your phone."
+- On a laptop:
+    * User navigates to example.com in a browser, sees an option to "Sign in with your phone."
     * User chooses this option and gets a message from the browser, "Please complete this action on your phone."
 
-- Next, on the phone:
+- Next, on their phone:
     * User sees a discreet prompt or notification, "Sign in to example.com."
     * User selects this prompt / notification.
     * User is shown a list of their example.com identities, e.g., "Sign in as Alice / Sign in as Bob."
-    * User picks an identity, is prompted for an authorization gesture (PIN, biometric, etc.) and provides this.
+    * User picks an identity, is prompted for an <a>authorization gesture</a> (PIN, biometric, etc.) and provides this.
 
-- Now, on the laptop:
-    * Web page shows that the selected user is signed in, and navigates to the signed-in page.
+- Now, back on the laptop:
+    * Web page shows that the selected user is signed-in, and navigates to the signed-in page.
 
 
-## Other configurations ## {#other-configurations}
+### Other use cases and configurations ### {#other-configurations}
 
 A variety of additional use cases and configurations are also possible, including (but not limited to):
 
-- User goes to example.com on their laptop, is guided through a flow to create and register a credential on their phone.
+- User navigates to example.com on their laptop, is guided through a flow to create and register a credential on their phone.
 
-- User employs a scoped credential as described above to authorize a single transaction, such as a payment or other financial
-    transaction.
+- A [RP] can cause a signed-in user to be prompted for their <a>authorization gesture</a> in order to authorize a single transaction, such as a payment or other financial transaction.
```

Comment:
Unnecessarily wordy. Also the user need not be signed in for the RP to authorize a transaction. Can we reduce to simply "A [RP] prompts the user for their authorization gesture..."? That would also match the voice used in the other bullet points.